terraform/helm/indexer/templates/_helpers.tpl

```diff
@@ -0,0 +1,63 @@
+{{/* vim: set filetype=mustache: */}}
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "aptos-indexer.name" -}}
```

Comment:
seems duplication with the fullname below

Reply:
It's autogenerated by helm :)